app/actions/handlers.py

```diff
@@ -49,6 +48,7 @@ async def action_pull_observations(
 
     # TODO: Create sub-actions for each destination
     total_observations = []
+    _client = GundiClient()
```

Comment:
I moved this down here to avoid weird issues with order of operations when pytest is loading fixtures and modules.